packages/presentation/src/component.ts

```diff
@@ -0,0 +1,278 @@
+//
```

Comment:
It should be defined in tracker plugin

Reply:
Also we need to think in future of how to start use of this pipeline only if tracker plugin is loaded, status support could be move to status plugins as well.